ViewControllers/ViewController.swift

```diff
@@ -47,9 +47,9 @@ class ViewController: UIViewController {
 			eventUrl  = "http://34.82.119.242/api/1.0"
 		} else {
 			envName = "UAT"
-			apiKey    = "5d59cb10-66cb-405b-ab54-b4d48132f383"
+			apiKey    = "ad1d75f6-0e94-4e34-b54c-db95b9325ae3"
 			configUrl = "https://config.feature-flags.uat.harness.io/api/1.0"
```

Comment:
can you ensure this SDK key is removed from the environment. we don't want keys to be available in the source code repo

Reply:
This is a part of the sample app.